lms/static/sass/views/_homepage.scss

```diff
@@ -49,7 +50,7 @@ $learn-more-horizontal-position: calc(50% - 100px); // calculate the left positi
 
         img {
           width: 100%;
-          height: auto;
+          min-height: $course-image-height;
```

Comment:
If I use a height, there's a slight chance it'll show as a thin band under images that are shorter than the area. I believe this only affects the demo course image, but I can't speak for every course image everywhere. Thoughts?